### DISCUSSION TITLE
bootstrap modals

### DISCUSSION THREAD

**Discussion by BilalKhatib** [twbs/bootstrap#37943] [Q&A] [answered]
### i have a question 
how can i open a bootstrap modal on an anchor tag click ? 

my code is like this : 
`<a href="owner-edit.php?owner_id='.$row['id'].'" data-bs-toggle="modal" data-bs-target="#EditOwnerModal">edit</a>`

the <a> tag is only changing the url and refreshing the page , im not getting the bootstrap modal 
even if i putted a button inside the <a> tag , it still don't work 

`<a href="owner-edit.php?owner_id='.$row['id'].'" data-bs-toggle="modal" data-bs-target="#EditOwnerModal"><button id="ownerEditModalBtn" type="button" data-id="'.$row['id'].'"  class="btn btn-warning w-50 editOwnerBtn" data-bs-toggle="modal" data-bs-target="#EditOwnerModal" name="editOwnerBtn"><span class="fa fa-pencil"></span></button></a>`

**Comment by denizgolbas** [comment]
What is your expectation. You click on an "a" tag and you go to that url. If you need to show modal, you should use button tag instead of a tag. This is not possible if you want the url you set to open in a modal view. You need to do some javascript manipulation.

**Reply by BilalKhatib** [reply]
i solved it , i had many modals conflicting with each others , that's the problem , thank you !

**Comment by BilalKhatib** [comment, marked as answer]
i solved it , i had many modals conflicting with each others , that's the problem , thank you !
